src/cljfx/ext/selection_model.clj

```diff
@@ -24,7 +24,7 @@
     (mutator/setter #(doto ^SelectionModel (get-model %1)
                        (.clearSelection)
                        (.select ^Object %2)))
-    lifecycle/scalar))
+    lifecycle/dynamic))
```

Comment:
This is a breaking change: all uses of `:selected-item` will have to be updated.